learn/what_is_meilisearch/comparison_to_alternatives.mdx

```diff
@@ -106,12 +107,25 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
 | Distinct attributes <br /><div style={{color: "#A9A9A9", fontSize: "0.9em"}}>De-duplicate documents by a field value</div>| ✅ | ✅ | ✅  | ✅ |
 | Grouping <br /><div style={{color: "#A9A9A9", fontSize: "0.9em"}}>Bucket documents by field values</div> | ❌ | ✅ | ✅  | ✅ |
 
+##### AI Search
```

Comment:
@mayasssh @macraig WDYT of that section name/grouping?

Reply:
I'd go for AI-powered search, but @mayasssh is the naming captain :)

Reply:
We had a discussion about this with the content team at the early stages of the GTM, and it was decided that 'AI-powered search' will be used as the general naming convention. For reference to hybrid search, I sometimes use 'AI-enhanced.